Week 7/Practice6.html

```diff
@@ -0,0 +1,17 @@
+    <!DOCTYPE html>
+    <html>
+        <head>
+            <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+        </head>
+        <body>
+            <p>My name is <span id="myName"></span></p>
+            <button>Click Me!</button>
+            <script>
+                const nameList = ["Shrek","Donkey","Princess Fiona"];
+                $("button").click(()=>{
+                var random = nameList[Math.floor(Math.random() * 3)];
```

Comment:
This should be a `const`. Watch for indentation and use `newList.length` to get the length of the array. Don't use the number 3 as the array size can change.